testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/DefaultConfigSmokeTestCase.java

```diff
@@ -123,7 +123,6 @@ private static void writeExpressionAudit(PathAddress pa, ModelNode resourceDescr
                     sb.append(",").append(attrdesc.get(ModelDescriptionConstants.TYPE).asString());
                     sb.append(",").append(attrdesc.get(ModelDescriptionConstants.EXPRESSIONS_ALLOWED).asBoolean(false));
                     sb.append(",").append(attrdesc.get(ModelDescriptionConstants.DESCRIPTION).asString());
-                    System.out.println(sb.toString());
```

Comment:
If this println is removed, sb variable is is useless and can be removed too, right? Wouldn't be better to replace this println with LOGGER.trace?